components/BottomNavigation/tests/unit/BottomNavigationThemingTests.m

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "../../src/private/MDCBottomNavigationItemView.h"
+#import "MaterialBottomNavigation+Theming.h"
+
+static const CGFloat kUnselectedOpacity = (CGFloat)0.6;
+@interface BottomNavigationThemingTests : XCTestCase
+@property(nonatomic, strong, nullable) MDCBottomNavigationBar *bottomNavigationBar;
+@property(nonatomic, strong, nullable) id<MDCContainerScheming> containerScheme;
+@end
+
+@interface MDCBottomNavigationBar (Testing)
+@property(nonatomic, strong) NSMutableArray<MDCBottomNavigationItemView *> *itemViews;
+- (void)applyThemeWithColorScheme:(id<MDCColorScheming>)colorScheme;
+- (void)applyThemeWithTypographyScheme:(id<MDCTypographyScheming>)typographyScheme;
+@end
+
+@interface MDCBottomNavigationItemView (Testing)
+@property(nonatomic, strong) UILabel *label;
+@end
+
+@implementation BottomNavigationThemingTests
```

Comment:
Does the new theming pass existing tests (to make sure there are no visual changes)?